xamarin/Samples/Showcase-Android/Assets/AboutAssets.txt

```diff
@@ -0,0 +1,19 @@
+Any raw assets you want to be deployed with your application can be placed in
```

Comment:
This file is automatically generated when you create a Xamarin solution. So...It's only information about how to use Assets. Should we keep this file or not?